lancet/commands/configuration.py

```diff
@@ -10,60 +10,101 @@
 
 
 @click.command()
-@click.option("-f", "--force/--no-force", default=False)
+@click.option(
+    "-f",
+    "--force/--no-force",
+    default=False,
+    help="Setup even if .lancet already exists.",
+)
+@click.option(
+    "-d",
+    "--debug/--no-debug",
+    default=False,
+    help="Adds additional steps to the wizard for debugging.",
+)
 @click.pass_context
-def setup(ctx, force):
-    """Wizard to create the user-level configuration file."""
-    if os.path.exists(USER_CONFIG) and not force:
+def setup(ctx, force, debug):
+    # environment variable used for testing
+    user_path = os.environ.get("ENV_USER_CONFIG") or USER_CONFIG
+
+    if os.path.exists(user_path) and not force:
         click.secho(
-            'An existing configuration file was found at "{}".\n'.format(
-                USER_CONFIG
-            ),
+            'An existing configuration file was found at "{}"'.format(user_path),
             fg="red",
             bold=True,
         )
         click.secho(
-            "Please remove it before in order to run the setup wizard or use\n"
-            "the --force flag to overwrite it."
+            "Please remove it before in order to run the setup wizard or use"
```

Comment:
The line breaks in the python code don't have to reflect the ones in the terminal. I'd undo this change.

Reply:
reverted